src/tests/mockData.js

```diff
@@ -0,0 +1,16 @@
+const users = {
+  success: {
+    email: 'Kamana@gmail.com',
+    password: 'kamana1234'
+  },
```

Comment:
Instead of mockData, let's look into seeding the database with data before we run the tests